### PR TITLE
Weak subjectivity

### LINKED ISSUE #576: Start from Weak Subjectivity State
Prepare Chainguardian to be able to start from WSS which skips most of sync and syncs just latest blocks.

Todo:
- service to fetch WSS either from Infura or BeaconScan
- boolean flag when configuring local node to let user decide if he want's to use it or not
- mount checkpoint data inside BN container and supply flag for wss start

Supported clients:
- Teku 
- need to check others

### PR DESCRIPTION
**Pull request by BeroBurny**
**Short description of work done**

### PR Checklist

- [x] I have run type check locally
- [x] I have run linter locally
- [x] I have run unit and integration tests locally
- [x] Rebased to master branch / merged master

### Issues

Closes #576

### CODE REVIEW

**Review by mpetrunic — DISMISSED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -30,7 +30,9 @@ jobs:
         run: yarn run test:unit && yarn run test:components
 
       - name: Run e2e tests
-        run: yarn run test:e2e
+        run: |
+          yarn add cheerio-select
```
Comment (mpetrunic) [inline .github/workflows/ci.yaml:34]: eh?

File: src/renderer/ducks/beacon/sagas.ts [hunk 2]
```diff
@@ -94,10 +99,82 @@ export function* pullDockerImage(
...
+        if (type === WeakSubjectivityCheckpoint.beaconScan) {
+            const httpClient = new HttpClient(`https://beaconscan.com/`);
+            if (network === "mainnet") {
+                const ws = yield httpClient.get(`ws_checkpoint`);
+                if (ws) return `${ws.ws_checkpoint}:${ws.current_epoch}`;
+            }
+
+            // TODO: change scraper with api after etherscan implement it
+            const dom = yield httpClient.get(network);
+            const home = cheerio.load(dom);
```
Comment (mpetrunic) [inline src/renderer/ducks/beacon/sagas.ts:135]: lol, we could have skipped this but ok 😅 
Could you write some e2e tests for this as they can change css at any point

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (BeroBurny) [reply .github/workflows/ci.yaml:34]: `--frozen-lockfile` dose not install `cheerio-select` even if is `cheerio-select` part of `package.json`

File: src/renderer/ducks/beacon/sagas.ts (hunk 2 above)
Comment (BeroBurny) [reply src/renderer/ducks/beacon/sagas.ts:135]: with cron like on dappateer?

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (mpetrunic) [reply .github/workflows/ci.yaml:34]: if it's in lockfile it must be installed

File: src/renderer/ducks/beacon/sagas.ts (hunk 2 above)
Comment (mpetrunic) [reply src/renderer/ducks/beacon/sagas.ts:135]: nah, just regular ones.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (BeroBurny) [reply .github/workflows/ci.yaml:34]: I have no idea why, maybe you can figure it out?

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**